cmake/FindSenseGlove.cmake

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Istituto Italiano di Tecnologia (IIT)
+# @author Kourosh Darvish <kourosh.darvish@iit.it>
+
+# Finds the Sense Glove SDK
+#
+# This will define the following variables::
+#
+#   SenseGlove_FOUND         - True if the system has the Sense Glove SDK
+#   SenseGlove_INCLUDE_DIR  - The include directory of the SDK
+#   SenseGlove_LIBRARIES     - The name of the libraries to link against.
+###############################
+
+# Check Directory of the SenseGlove_DIR
+if(NOT DEFINED ENV{SenseGlove_DIR})
+  message( FATAL_ERROR "Environment variable {SenseGlove_DIR} is not defined: $ENV{SenseGlove_DIR}" )
+else()
+  message("Environment variable {SenseGlove_DIR}: $ENV{SenseGlove_DIR}" )
+endif()
+
+if(NOT DEFINED ENV{SenseGlove_INCLUDE_DIRS})
+  message( FATAL_ERROR "Environment variable {SenseGlove_INCLUDE_DIRS} is not defined: $ENV{SenseGlove_INCLUDE_DIRS}" )
+else()
+  message("Environment variable {SenseGlove_INCLUDE_DIRS}: $ENV{SenseGlove_INCLUDE_DIRS}" )
+endif()
+##### Find SenseGlove #####
+
+add_library(SenseGlove SHARED IMPORTED GLOBAL $ENV{SenseGlove_DIR}/libSGCoreCpp.so)
+set_target_properties(SenseGlove PROPERTIES IMPORTED_LOCATION $ENV{SenseGlove_DIR}/libSGCoreCpp.so)
+target_include_directories(SenseGlove INTERFACE $ENV{SenseGlove_INCLUDE_DIRS})
+
+set(SenseGlove_FOUND TRUE)
+
+
+
```

Comment:
cc @traversaro, I'm not expert in developing `Find*.cmake` files.

Reply:
It seems ok, the only question is if there is some docs somewhere on how the SenseGlove SDK needs to be installed and used? Because the only thing different from the usual `Find**.cmake`  workflow is that the `CMAKE_PREFIX_PATH`  is not searched for libraries and headers files, but rather `SenseGlove_DIR` env variable **must** be defined.

Reply:
>  if there is some docs somewhere on how the SenseGlove SDK needs to be installed and used?

Regarding this point, a minimum information should be added in the `README.md` at https://github.com/robotology/wearables#wearable-device-sources

Reply:
> It seems ok, the only question is if there is some docs somewhere on how the SenseGlove SDK needs to be installed and used? Because the only thing different from the usual Find**.cmake workflow is that the CMAKE_PREFIX_PATH is not searched for libraries and headers files, but rather SenseGlove_DIR env variable must be defined.

What I was thinking to do, is later fork this repo https://github.com/Adjuvo/SenseGlove-API to DIC organization. Later, there are two environment variables to set.

Reply:
@traversaro  Note that, I am using currently two environment variables for the senseGlove, i.e., `SenseGlove_DIR` and `SenseGlove_INCLUDE_DIRS`. Moreover, `SenseGlove_DIR` is passed to `PATH` as well. If this is OK, I resolve this conversation.

Reply:
Can't we put this info in the README as suggested by @lrapetti ? If this takes time no problem, but if it is fast is probably worth to do it!

Reply:
I am adding a README to the HapticGlove Floder folder

Reply:
Thanks! Probably it could make sense to link it from the main README, otherwise it may be difficult to locate.

Reply:
I have added the README to the device and also to the https://github.com/robotology/wearables#wearable-device-sources

Let me know if it is OK.

Reply:
Thanks, @kouroshD. Just a small comment, in order to be coherent, maybe you could move it in [`doc`](https://github.com/robotology/wearables/tree/master/doc) together with the documentation of the other devices. Eventually you can call it `how-to-compile-and-run...`

Reply:
Can I resolve this conversation, I have updated with the requested changes?

Reply:
Ok for me.